data/plugins.json

```diff
@@ -157,23 +206,33 @@
     "safari": 10,
     "node": 6,
     "ios": 10,
+    "chrome-flag": null,
+    "edge-flag": 13,
+    "node-flag": 6,
     "opera": 37
   },
   "transform-exponentiation-operator": {
     "chrome": 52,
     "edge": 14,
     "firefox": 52,
+    "chrome-flag": 51,
+    "edge-flag": 14,
+    "node-flag": 6.5,
     "opera": 39
   },
   "transform-async-to-generator": {
     "chrome": 55,
     "firefox": 52,
+    "chrome-flag": 52,
+    "edge-flag": 13,
     "opera": 42
   },
   "syntax-trailing-function-commas": {
```

Comment:
Ideally I'd like to keep the trailing comma, both for technical purity reasons and for making the generated code as close as possible to what's written